src/Core/Logging/Extensions/ILogServiceExtensions.Info.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceExtensions
```

Comment:
Codacy found an issue: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5762132)